### PR TITLE
Switch to generic version for helm upgrade

### PR DESCRIPTION
**Pull request by AaronCrawfis**

### CODE REVIEW

**Comment by Reshrahim** [comment]
@AaronCrawfis - shouldn't we change here under helm tab as well (https://radapp.dev/operations/platforms/kubernetes/#install-the-radius-control-plane)

File: docs/content/operations/providers/_index.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ The Azure provider allows you to deploy and connect to Azure resources from a se
    ```
 1. Install the Radius Helm chart with the azure provider values set:
    ```bash
-   helm upgrade radius radius/radius --install --create-namespace --namespace radius-system --version 0.12.0 --wait --timeout 15m0s --set rp.provider.azure.podidentity=radius --set rp.provider.azure.subscriptionId=${MY_SUBSCIRPTION_ID} --set rp.provider.azure.resourceGroup=${MY_RESOURCE_GROUP}
+   helm upgrade radius radius/radius --install --create-namespace --namespace radius-system --version <VERSION> --wait --timeout 15m0s --set rp.provider.azure.podidentity=radius --set rp.provider.azure.subscriptionId=${MY_SUBSCIRPTION_ID} --set rp.provider.azure.resourceGroup=${MY_RESOURCE_GROUP}
```
Comment (Reshrahim) [inline docs/content/operations/providers/_index.md]: How would the user know the version?

**Comment by AaronCrawfis** [comment]
> @AaronCrawfis - shouldn't we change here under helm tab as well (https://radapp.dev/operations/platforms/kubernetes/#install-the-radius-control-plane)

So I actually forget I parameterized this awhile back with `{{< param chart_version >}}` 😅. Updated PR to use that instead of `VERSION`. Thanks for catching that!

**Review by Reshrahim — CHANGES_REQUESTED** [review]

File: docs/content/operations/providers/_index.md (hunk 1 above)
Comment (AaronCrawfis) [reply docs/content/operations/providers/_index.md]: I believe this comment was on an outdated version. I've switched it to use the parameter specified in `config.toml` which renders to the latest version

Comment (Reshrahim) [reply docs/content/operations/providers/_index.md]: I see the answer to this pushed i the new commit

**Review by Reshrahim — APPROVED** [review]

**Pull request merged**